external-builds/pytorch/pytorch_utils.py

```diff
@@ -137,28 +134,23 @@ def get_all_supported_devices(
         original_system_indices = [
             int(idx.strip()) for idx in current_hip_visible.split(",")
         ]
-        if log:
-            print(f"HIP_VISIBLE_DEVICES already set to: {current_hip_visible}")
+        print(f"HIP_VISIBLE_DEVICES already set to: {current_hip_visible}")
     else:
         # HIP_VISIBLE_DEVICES not set, no remapping needed
         original_system_indices = None
 
     # Query both supported and visible GPUs in a single subprocess call
     # (doesn't initialize CUDA in main process)
-    if log:
-        print("Getting GPU information from PyTorch...", end="")
+    print("Getting GPU information from PyTorch...")
     supported_gpus, raw_visible_gpus = get_supported_and_visible_gpus()
-    if log:
-        print("done")
```

Comment:
please readd the `print("done")`. as this is started in a subprocess it takes a bit of time